backend/src/scripts/addFakeData.ts

```diff
@@ -1,5 +1,5 @@
 import { findAndUpdateUser } from '../services/user.js'
-import { connectToMongoose, disconnectFromMongoose } from '../utils/database.js' assert { type: 'json' }
```

Comment:
This has always been needed before, is there a reason we don't need it anymore?

Reply:
We do need it on JSON files, but `../utils/database.js` is not a JSON file.